src/memfs.rs

```diff
@@ -1280,11 +1280,11 @@ impl Filesystem for MemoryFilesystem {
         );
 
         let read_helper = |content: &Vec<u8>| {
-            if (offset as usize) < content.len() {
-                let read_data = if ((offset + size as i64) as usize) < content.len() {
-                    &content[(offset as usize)..(offset + size as i64) as usize]
+            if offset.cast::<usize>() < content.len() {
+                let read_data = if (offset + size.cast::<i64>()).cast::<usize>() < content.len() {
+                    &content[offset.cast()..(offset + size.cast::<i64>()).cast()]
```

Comment:
offset.cast() + size
no need extra cast()

Reply:
The same

Reply:
`offset.cast::<usize>() + size.cast::<usize>()`

Reply:
Done